iothub/device/src/ITransportSettings.cs

```diff
@@ -20,5 +21,10 @@ public interface ITransportSettings
         /// The default receive timeout.
         /// </summary>
         TimeSpan DefaultReceiveTimeout { get; }
+
+        /// <summary>
+        /// The proxy settings to be used for communication.
+        /// </summary>
+        IWebProxy Proxy { get; set; }
```

Comment:
`Proxy` is already defined in each of the transport settings that implement `ITransportSettings`. I am defining it in the interface as well, so that it can be easily initialized and tested.
This does not change the API surface.